scripts/flattener.py

```diff
@@ -433,6 +447,32 @@ def gather_pooled_metadata(obj_type, properties, values_to_add, objs):
 				values_df[i] = 'unknown'
 			for index, row in values_df.iterrows():
 				values_to_add[index] = 'pooled [{}]'.format(','.join(row.to_list()))
+		elif prop == 'ethnicity':
+			values_df = pd.DataFrame()
+			unknowns = []
+			for obj in objs:
+				ident = obj.get('@id')
+				ethnicity_list = []
+				ethnicity_dict_list = get_value(obj,prop)
+				if ethnicity_dict_list != None:
+					for ethnicity_dict in ethnicity_dict_list:
+						if ethnicity_dict.get('term_id') == 'NCIT:C17998':
+							ethnicity_list.append('unknown')
+						else:
+							ethnicity_list.append(ethnicity_dict.get('term_id'))
+				if len(ethnicity_list) == 1 and ethnicity_list[0] == 'unknown':
+					values_df[ident] = np.nan
+					unknowns.append(ident)
+				else:
+					ethnicity_list.sort()
+					value = ','.join(ethnicity_list)
+					latkey = (obj_type + '_' + prop).replace('.','_')
+					key = prop_map.get(latkey, latkey)
+					values_df.loc[key,ident] = value
+			for i in unknowns:
+				values_df[i] = 'unknown'
+			for index, row in values_df.iterrows():
+				values_to_add[index] = 'pooled [{}]'.format(','.join(row.to_list()))
```

Comment:
For LATDF477OUM, this is returning `pooled [HANCESTRO:0005,HANCESTRO:0005]` for self_reported_ethnicity_ontology_term_id. Was there a specific ProcMatrixFile that you were targeting this format for?
This is a little tricky. I think for these, where there are samples are multiplexed but not computationally de-multiplexed, if there ends up being a list of ethnicities, the actual ethnicity should be `unknown`. @jahilton Thoughts? A single cell from that multiplexed sample can be from either donor. And if the donors are of different ethnicities, there is no way to determine which ethnicity that cell is. 
So, for this code, I think first check uniqueness of list. And then, if this pooled sample is still of mixed ethnicities, then set to `unknown`.

Reply:
So we're thinking of the case where a cell is from multiple donors?
Ideally, if the ethnicities match completely amongst those donors, then we use those (whether it's one or a list)
If there is any different amongst them, then we should go to `unknown`
But if the comparison is too onerous, then just pooling --> `unknown` always would be acceptable, I think

Thanks for flagging that case